SonarQube.Common/VerbosityCalculator.cs

```diff
@@ -61,28 +61,6 @@ public static LoggerVerbosity ComputeVerbosity(IAnalysisPropertyProvider propert
             return ComputeVerbosity(sonarVerboseValue, sonarLogLevelValue, logger);
         }
 
-        /// <summary>
-        /// Computes verbosity based on the analysis config file settings. 
-        /// </summary>
-        /// <remarks>If no verbosity setting is present, the default verbosity (info) is used</remarks>
-        public static LoggerVerbosity ComputeVerbosity(AnalysisConfig config, ILogger logger)
```

Comment:
This method is no longer required: we can use the other "ComputeVerbosity" method that accepts an IAnalysisPropertyProvider instead.